src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$primary-color-black: #060b35;
+$primary-color-grey: #616070;
+$primary-color-blue: #00acdc;
+$primary-color-white: #fff;
```

Comment:
if u always use the prefix primary.. mb we can simplify it and use just $color- + color name?)